prow/tide/tide.go

```diff
@@ -822,11 +822,19 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 	for i, pr := range prs {
 		log := log.WithFields(pr.logFields())
 		mergeMethod := c.config().Tide.MergeMethod(sp.org, sp.repo)
-		if squashLabel := c.config().Tide.SquashLabel; squashLabel != "" {
+		squashLabel := c.config().Tide.SquashLabel
+		rebaseLabel := c.config().Tide.RebaseLabel
+		mergeLabel := c.config().Tide.MergeLabel
+		if squashLabel != "" || rebaseLabel != "" || mergeLabel != "" {
 			for _, prlabel := range pr.Labels.Nodes {
```

Comment:
It is possible for a PR to have both a squash, rebase and merge label. It seems like we should detect this scenario and error out

Reply:
I think this increases the complexity a bit, but took a shot at trying to do this in the latest commit.